src/cljs/main/broadfcui/auth.cljs

```diff
@@ -308,14 +308,9 @@
                         (str "Could not load Terms of Service; please read it at "
                            (str (config/terra-base-url) "/#terms-of-service")
                            ".")))))
-              (case status-code
-                ;; 403 means the user declined the TOS (or has invalid token? Need to distinguish)
-                403 (swap! state assoc :error :declined)
-                ;; 404 means the user hasn't seen the TOS yet and must agree (or url is wrong? need to distinguish)
-                404 (swap! state assoc :error :not-agreed)
-                (swap! state assoc :error (handle-server-error status-code get-parsed-response)))))))))})
+              (swap! state assoc :error :not-agreed)))))))})
 
-(defn reject-tos [on-done] (endpoints/tos-set-status false on-done))
+(defn reject-tos [on-done] (endpoints/tos-set-status "tos-rejected" on-done))
```

Comment:
does this do anything on the back end? What does sending an intentionally incorrect tos url to Sam actually do?

Reply:
That's a good point, this should be a no-op